sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/auth/opa/OpaAuthorizationManager.java

```diff
@@ -79,11 +77,11 @@ public OpaAuthorizationManager(
       @Value("${opa.policy.package:}") String policyPackage,
       OpaRequestBuilder opaRequestBuilder,
       @Qualifier("opaRestTemplate") RestTemplate opaRestTemplate,
-      ApplicationContext applicationContext
-      /*, TODO update to opentelemetry Object tracer*/ ) {
+      ApplicationContext applicationContext,
+      OpenTelemetry openTelemetry) {
```

Comment:
NIT: We removed the `@param` in the JavaDoc when it was not available. Can you add the docs again?